src/ClientGenerator/ClientGenerator.cs

```diff
@@ -102,6 +103,28 @@ private static bool CreateGrainClientAssembly(CodeGenOptions options)
         }
 
         /// <summary>
+        /// Configure logger factory for your client generator
+        /// TODO: fix logging in ClientGenerator, I'm not sure its logging worked properly before, taken that LogManager's state doesn't serialize well across AppDomain
+        /// </summary>
+        public ILoggerFactory LoggerFactory { get; set; } = CreateDefaultLoggerFactory("ClientGenerator.log");
+
+        /// <summary>
+        /// Create the default logger factory, which would create <see cref="Microsoft.Extensions.Logging.ILogger"/>> that writes logs to <paramref name="filePath"/> and console.
+        /// This is to restore legacy default behavior of LogManager, which configure LogManager with a FileTelemetryConsumer and ConsoleTelemetryConsumer
+        /// by default;
+        /// </summary>
+        /// <param name="filePath">the logger file path</param>
+        /// <returns></returns>
+        private static ILoggerFactory CreateDefaultLoggerFactory(string filePath)
+        {
+            var factory = new LoggerFactory(new List<ILoggerProvider>());
```

Comment:
Is the `new List<ILoggerProvider> ()` needed here?

Reply:
not needed.  a mistake when copied from other CreateDefaultLoggerFactory method when it can put in filters param